### PR TITLE
Adds roburger crafting recipe for tators

### PR DESCRIPTION
**Pull request by ghost**
# Document the changes in your pull request

Adds a "Roburger recipe" item for traitors. It is a hijack/martyr only item, and costs 14 TC, meaning that you can still buy an emag to make this item have an actual usage. Also since i am not a spriter it has the same icon as makeshift items 101, so if you can help me with the sprite i will be thankfull.

Roburger recipe requires a bun, a borg head and 10 cable coils. It isn't available by normal condition. 

# Wiki Documentation

Add roburger recipe into guide to food, and add the recipe item into syndicate gear.

# Changelog

Edit the changelog below to reflect the changes made by this PR, even if the changes are minor - required for every PR.
If you add a name after the ':cl', that name will be used in the changelog. Leave it empty to use your GitHub name.

:cl:  
rscadd: Adds a roburger recipe for tators
/:cl:

### CODE REVIEW

**Comment by ynot01** [comment]
maybe a bit OP but requires some creative application and since it's only hijack/martyr it shouldn't be abused often

**Comment by ghost** [comment]
> maybe a bit OP but requires some creative application and since it's only hijack/martyr it shouldn't be abused often

Yeah

**Comment by bruhlookatthisdood** [comment]
can we know what it does?

**Comment by ghost** [comment]
> can we know what it does?

Uhhh... turns people into borgs. It is an old food, that didn't had a crafting recipe.

**Comment by bruhlookatthisdood** [comment]
like instantly?

**Comment by ghost** [comment]
> like instantly?

I don't know, I didn't create roburgers. As i know they contain nanobots reagent, that infects people with a borg transformation disease.

**Comment by adamsong** [comment]
You get disease, when disease reaches stage 5 (not sure how long, I'm not a mentor), become borg

File: code/modules/uplink/uplink_items.dm [hunk 1]
```diff
@@ -1537,6 +1537,16 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
...
 	exclude_modes = list(/datum/game_mode/infiltration) // yogs: infiltration
 
+/datum/uplink_item/device_tools/roburger_recipe
+	name = "Roburger crafting recipe"
+	desc = "This book contains legendary knowledge about how to make roburgers. \
+			Roburgers turn people into cyborgs after a short while when eaten. \
+			This book doesn't dissaper after use, so consider hiding it somewhere."
+	item = /obj/item/book/granter/crafting_recipe/roburgers
+	cost = 14
+	include_objectives = list(/datum/objective/hijack, /datum/objective/martyr)
```
Comment (adamsong) [inline code/modules/uplink/uplink_items.dm]:
```suggestion
	include_objectives = list(/datum/objective/hijack, /datum/objective/martyr, /datum/objective/nuclear)
```

**Review by adamsong — DISMISSED** [review]

**Comment by maxion12345** [comment]
Not sure about this. You can get them (and other wacky food) from xenobio silver slimes, so I'm unsure if this will be a popular purchase.

**Review by JamieD1 — APPROVED** [review]
Lets seeee

**Comment by Reed0506** [comment]
letting antags have access to niche gimmick items with little to no way to normally obtain them is very good

**Comment by ghost** [comment]
> Not sure about this. You can get them (and other wacky food) from xenobio silver slimes, so I'm unsure if this will be a popular purchase. 

Not anyone can do/has access to xenobio

**Comment by Lovebird-Frungy** [comment]
Would be interesting to add nanite heart to the recipe.
Its hard to obtain, yes, but its on theme with the nanite virus thing.

**Comment by ghost** [comment]
> Would be interesting to add nanite heart to the recipe.
> 
> Its hard to obtain, yes, but its on theme with the nanite virus thing.

I don't think that adding nanite heart to the recipe will be a good idea. As you said it is hard to obtain, so it wouldn't be the best choice.

**Pull request merged**